### DISCUSSION TITLE
GPU 问题  & 注释问题

### DISCUSSION THREAD

**Discussion by wujun007** [SciSharp/TensorFlow.NET#1103] [Q&A] [answered]
首次使用TensorFlow.net  框架，总体感觉来说还是挺不错的，但也暴露了两个问题
1.整套框架下的文档注释零零散散，对于开发者来说确实不太友好，希望早日追加注释，这个显得非常重要。
2.在测试这个框架性能的时候发现GPU版本中，在计算时显存确实在提高，但GPU 并没有进行运算，就想是走了一个过场，将当前要运算的张量在GPU的显存中复制了一下，然后就结束了，主要计算还是在CPU上，期望下一个版本这些问题都能得到改善。
       

                                                                    祝：希望这个项目能一直走下去，这个项目的前途将无可限量。

**Comment by Oceania2018** [comment, marked as answer]
希望你能更深入的参与到项目中提供更有价值的参考意见。

**Reply by wujun007** [reply]
请问你们有讨论的QQ技术群或者微信群吗？这样的话可以更加方便的进行技术交流，并且我提到的这个GPU的问题，测试代码我这里有，可以重现我所述的这个问题

**Reply by Wanglongzhi2001** [reply]
qq群:985366726

**Reply by Oceania2018** [reply]
点主页的这个图标就可以加群：

![image](https://github.com/SciSharp/TensorFlow.NET/assets/1705364/9d1f00ab-1179-4da3-b682-b723466125ef)
